ui/components/Footer.tsx

```diff
@@ -16,7 +17,34 @@ const RightFoot = styled(Flex)`
 
 const LeftFoot = styled(Flex)``;
 
+const REPO_URL = "https://github.com/weaveworks/weave-gitops";
+
 function Footer({ className }: Props) {
+  const { data, isLoading } = useVersion();
+  const versionData = data?.version || {} as Version;
+  const apiVersion: Version = {
+    version: versionData.version,
+    gitCommit: versionData["git-commit"],
```

Comment:
I don't think this object is necessary.

We should change the backend to return a better `JSON` key: 

https://github.com/weaveworks/weave-gitops/blob/3d1af824228d8f2974003a8489be4940267a62d1/core/server/version.go#L16

We can enumerate the keys here and have `protoc` generate the types for us:

https://github.com/weaveworks/weave-gitops/blob/3d1af824228d8f2974003a8489be4940267a62d1/api/core/core.proto#L349

Happy to help on this.

Reply:
Fixed this